index.js

```diff
@@ -17,7 +17,11 @@ try {
     core.exportVariable('SSH_AUTH_SOCK', authSock);
 
     console.log("Adding private key to agent");
-    child_process.execSync('ssh-add -', { input: core.getInput('ssh-private-key') });
+    core.getInput('ssh-private-key').split(/(?=-----BEGIN)/).forEach(function(key) {
```

Comment:
@mpdude why didn't you add an end-marker here? It'd be way more precise than trimming `\n` this way... People would even be able to put all the keys in one line if they wanted to